app/app/vhosts/new/route.js

```diff
@@ -39,6 +40,8 @@ export default Ember.Route.extend({
         return op.save();
       }).then( () => {
         let message = `Domain ${vhost.get('virtualDomain')} created`;
+        app.get('vhosts').unshiftObject(vhost);
```

Comment:
You're right, this unshift is unnecessary with the ED relationships set up correctly.  Removing...